healthyregions/chicago-environment-explorer
# feat: add Impervious Surfaces as a new overlay

## Problem
We have received another submission for a new overlay and would like to display that on the ChiVes map

Fixes #187 

## Approach
* feat: added Impervious Surfaces overlay to the ChiVes map
    * Displays Airports, Major Roadways, and Industrial Areas
    * Open question: does this overlap with Non-Residential/Industrial overlay?

![Screenshot 2024-08-13 at 12 59 14 PM](https://github.com/user-attachments/assets/ebe6d15e-c55c-40d6-87a3-ea3bb79e5d96)

## How to Test
1. Navigate to https://deploy-preview-188--chicago-env-explorer.netlify.app/map
2. In the left panel, scroll down to the Overlays dropdown
3. De-select all overlay, then select "Impervious Surfaces"
    * You should see the dark/transparent overlay appear on the map
